core/src/main/java/org/mobilitydata/gtfsvalidator/notice/DuplicateKeyError.java

```diff
@@ -39,7 +39,7 @@ public DuplicateKeyError(
             "newCsvRowNumber",
             newCsvRowNumber,
             fieldName,
-            fieldValue));
+            fieldValue), severityLevel);
```

Comment:
This should be `SeverityLevel.ERROR`.

In other words, when the developer is creating a new notice that `extends ValidationNotice` they should define the severity level within the Notice implementation. The severity level shouldn't be exposed in the constructor parameters of the notice unless the `*Validator` class needs to sometimes declare it an `ERROR` and sometimes a `WARNING` based on context. Same for all other notices that `extends ValidationNotice`.